File: C++/Estrutura sequencial/pagamento.cpp
// Biblioteca
#include <bits/stdc++.h>

using namespace std;

int main()
{
    // Variáveis
    string nome;
    double valorHora, pagamento;
    int horaTrabalhada;

    // Entrada de dados
    cout << "Nome: ";
    getline(cin, nome);

    cout << "Valor por hora: ";
    cin >> valorHora;

    cout << "Horas trabalhadas: ";
    cin >> horaTrabalhada;

    // Algoritmo matemático
    pagamento = (double) valorHora * horaTrabalhada;

    // Saída de dados
    cout << fixed << setprecision(2); // Quantidade de casas decimais
    cout << "O pagamento para " << nome << " deve ser R$" << pagamento;

    return 0;
}

File: Python/Estrutura sequencial/pagamento.py
# Variáveis
# Em Python não é necessário fazer esse passo...
nome: str
valorHora: float; pagamento: float
horasTrabalhadas: int

# Entrada de dados
nome = input("Nome: ")
valorHora = float(input("Valor por hora: "))
horasTrabalhadas = int(input("Horas trabalhadas: "))

# Artíficio matemático
pagamento = valorHora * horasTrabalhadas

# Saída de dados
print(f"O pagamento para {nome} deve ser {pagamento}")